src/Orchard/ContentManagement/IContentManager.cs

```diff
@@ -42,6 +42,14 @@ public interface IContentManager : IDependency {
         ContentItem Clone(ContentItem contentItem);
 
         /// <summary>
+        /// Makes a clone of the content item with a special context
+        /// </summary>
+        /// <param name="contentItem">The content item to clone</param>
+        /// <param name="cloneContext">The cloning context</param>
+        /// <returns>Clone of the item</returns>
+        ContentItem Clone(ContentItem contentItem, Handlers.CloneContentContext cloneContext);
```

Comment:
This implementation is used by Localization. It's needed for Translate (Localization), which uses `Clone` with a `TranslateContentContext` (derived from `CloneContentContext`).